### PR TITLE
blog: add multi-level vmemcache usage

### PR DESCRIPTION
**Pull request by kilobyte**
This change is [Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/102)

### CODE REVIEW

**Comment by kilobyte** [comment]
@pbalcer @ldorau @George330 please review.

**Comment by pbalcer** [comment]
Can you setup `kilobyte.github.io` so that we can see a rendered page?

File: _posts/2019-05-16-two-level-vmemcache.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+them into such **vmemcache** instances.  But, if the cache is not static,
+technique from this post may be of use to you.
+
+### Tiers
+
+While **vmemcache** has been optimized for byte-addressable storage (DRAM or
+Optane™ DC Persistent Memory), it can also work on block-based media¹, no
+matter how it is accessed (NVDIMM-F, NVMe, SATA, network,
+[IPoAC](https://en.wikipedia.org/wiki/IP_over_Avian_Carriers)).  This allows a
+mix of different kinds to be used by the same program.  In particular,
```
Comment (wlemkows) [inline _posts/2019-05-16-two-level-vmemcache.md]: double spaces everywhere

File: _posts/2019-05-16-two-level-vmemcache.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
+---
+title: two-level vmemcache
```
Comment (marcinslusarz) [inline _posts/2019-05-16-two-level-vmemcache.md]: Multi-level vmemcache?

**Comment by kilobyte** [comment]
@George330: ping.

**Pull request merged**